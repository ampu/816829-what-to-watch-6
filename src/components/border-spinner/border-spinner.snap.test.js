import React from 'react';

import {render} from '@testing-library/react';

import BorderSpinner from './border-spinner';

describe(`BorderSpinner component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<BorderSpinner/>);
    expect(container).toMatchSnapshot();
  });
});
