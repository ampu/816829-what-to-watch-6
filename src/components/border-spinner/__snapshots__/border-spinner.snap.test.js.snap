// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BorderSpinner component Should render according to the snapshot 1`] = `
<div>
  <div
    class="border-spinner"
  />
</div>
`;
